openmdao/docs/features/recording/iterating_hierarchically.rst

```diff
@@ -0,0 +1,42 @@
+***********************************
+Iterating Over Case Hierarchies
+***********************************
+
+The Case Reader offers a number of ways to iterate over cases. Perhaps
+the simplest is using the :code:`list_cases` method with :code:`get_case`
+to iterate over all cases of a certain type.
+
+.. code-block:: console
+
+    case_keys = cr.driver_cases.list_cases()
+    for case_key in case_keys:
+        case = cr.driver_cases.get_case(case_key)
+        ...
+
+The Case Reader also has a :code:`get_cases` method, which provides a way to iterate
+over Driver and Solver cases in order with a choice of iterating hierarchically or flat
```

Comment:
"a choice of iterating either hierarchically or flaty" ?  What do you think? "Iterating in either a hierarchical or a flat manner" ?